staging/kos/cmd/attachment-tput-driver/main.go

```diff
@@ -1126,10 +1132,18 @@ func selectSlots(thd, numThreads int, from []VirtNetAttachment) []VirtNetAttachm
 func RunThread(kClientset *kosclientset.Clientset, stopCh <-chan struct{}, work []VirtNetAttachment, nodes []k8sv1.Node, nattNameFmt, runID string, opsTimes OpsSchedule, tbase time.Time, numAttsToCreate, thd, numThreads uint64, roundRobin, justCreate bool) {
 	glog.Warningf("Thread start: thd=%d, numAttachments=%d, stride=%d\n", thd, numAttsToCreate, numThreads)
 	var iCreate, iDelete uint64
+	var lateOps uint64
+	var opsDelay time.Duration
+	defer func() {
+		atomic.AddUint64(opsCompleted, iCreate+iDelete)
+		atomic.AddUint64(totalLateOps, lateOps)
+		atomic.AddInt64(totalOpsDelayNanos, int64(opsDelay))
+	}()
```

Comment:
There's probably a more elegant solution than this. It consists of replacing the wait group that the main thread uses to wait for threads that issue requests with buffered channels that the workers use to send the variables in the code this comment is about. But for now this will do.

Reply:
What's here is fine.

Reply:
Ok, let's leave it unchanged.